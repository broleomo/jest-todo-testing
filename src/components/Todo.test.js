import React from 'react';
import renderer from 'react-test-renderer';
import Todo from "./Todo";

test('Rendering correctly', () => {
  const tree = renderer.create(
    <Todo todo={{text:"Test", id:1, done: false}}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})
