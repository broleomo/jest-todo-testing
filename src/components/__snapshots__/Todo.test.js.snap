// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering correctly 1`] = `
<div
  className="Todo"
>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox"
  />
  <span
    className="todo-not-done"
    onClick={[Function]}
  >
    Test
  </span>
</div>
`;
